docs/nfts.md

```diff
@@ -2,4 +2,79 @@
 title: NFTs
 ---
 
-Docs coming soon.
+### React function component example
+
+The following is a component that implements various tool examples using a previous json dump named `ConsolidatedRemarks` (with the working code equivalent commented out). In a nutshell, the component logs the winners of my contest [bishoujo.store](http://bishoujo.store) with a counter of how many times that individual emoted on the nft.
+
+```
+// const RMRK_PREFIXES = ['0x726d726b', '0x524d524b'];
+const collectionId = '0488b1a94fd8d61738-BISHOUJO';
+export const NftCollection: React.FC = () => {
+  React.useEffect(() => {
+    // const provider = new WsProvider('wss://node.rmrk.app');
+    const fetchAndConsolidate = async () => {
+      try {
+        // const api = await ApiPromise.create({ provider });
```

Comment:
Commented parts of code are just confusing. Focus on the logic you're demonstrating only. Explain why, what it does, and what the point of this example is (what the reader learns).